components/TextInput.js

```diff
@@ -0,0 +1,40 @@
+import propTypes from 'prop-types'
+
+export default function TextInput(props) {
+  return (
+    <div className="mt-6">
+      <label
+        htmlFor={props.id}
+        className="block rounded-t-lg overflow-hidden border-t border-l border-r border-gray-300 px-3 py-2 bg-gray-300 justify-cente"
```

Comment:
```suggestion
        className="block rounded-t-lg overflow-hidden border-t border-l border-r border-gray-300 px-3 py-2 bg-gray-300 justify-center"
```
small typo here, though I don't think  justify-center actually does anything here since you're using `block` instead of `flex`

Reply:
Oh, yea I do that sometimes, I add classes and then forget to remove them if I am not using them. I will double-check everything